examples/runDecamQuickTest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+print_error() {
```

Comment:
`print_error` is repeatedly defined but not used in any of these scripts except `runExample.sh`.